nightly/expensive.txt

```diff
@@ -29,7 +29,7 @@ expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_
 expensive --timeout=1800 near-client near_client tests::catching_up::test_chunk_grieving --features nightly_protocol,nightly_protocol_features
 
 expensive integration-tests test_catchup test_catchup
-expensive integration-tests test_catchup test_catchup --features nightly_protocol,nightly_protocol_features
+expensive integration-tests test_catchup tests::test_catchup::test_catchup --features nightly_protocol,nightly_protocol_features
```

Comment:
You will also need to change the third column from `test_catchup` to `integration_tests` I believe. Same in all other lines.